server/issue.go

```diff
@@ -673,6 +673,11 @@ func (p *Plugin) assignJiraIssue(mmUserId, issueKey, assignee string) (string, e
 	// user is array of one object
 	user := jiraUsers[0]
 
+	// From Jira error: query parameters 'accountId' and 'username' are mutually exclusive.
```

Comment:
This should be preferring AccountId, but defaulting to Name to cover the Jira Server case?

Reply:
Maybe we should be testing what install we're on for this. `if ji.GetType() == JIRATypeCloud` etc.

Reply:
I'm down for any hack for 2.1 for this. The future really ought to be a `type User interface { ... Key() string ... }`

Reply:
I went with the quick fix.